i18n/sync-i18n.js

```diff
@@ -109,6 +113,44 @@ function remove_linebreaks(str) {
     return str.replace(/[\r\n]+/gm, " ");
 }
 
+// Remove from existing & valid keys in the localizedVersion from the base version; 
+function compareKeyHashes(baseMessages, localizedMessages) {
+
+  baseMessages = { ...baseMessages }; // copy
+
+  const meta = localizedMessages['#meta']
+  if(meta?.hashes){
+    delete localizedMessages['#meta'];
+
+    Object.entries(localizedMessages).filter(([, value]) => !!value)
+      .forEach(([key]) => {
+        
+          //if the key has hash only then continue with locale else use english
+          if(baseMessages[key] && meta.hashes[key]){
+            let enHash;
+            const enText = baseMessages[key];
+            
+            if(meta.algorithm =='md5'){
+              enHash = crypto.createHash('md5').update(enText).digest("hex");
```

Comment:
do you need to pass encoding for update?

Reply:
No default to utf-8 https://nodejs.org/api/crypto.html#hashupdatedata-inputencoding